src/layouts/Layout.astro

```diff
@@ -0,0 +1,37 @@
+---
+import GlobalStyles from '../components/GlobalStyles.astro';
+
+interface Props {
+	currentPage: string;
+}
+
+const { currentPage }: Props = Astro.props;
+---
+
+<html>
+	<head>
+		<meta name='viewport' content='width=device-width, initial-scale=1.0' />
+		<GlobalStyles />
+		<slot name='head' />
+		<title>{currentPage}</title>
+	</head>
+	<body>
+		<header>
+			<h1 class='accent'>
+				Redstone.Tools{currentPage === 'Home' ? '' : ` - ${currentPage}`}
```

Comment:
is "Redstone.Tools" supposed to be "Redstone Tools"?

Reply:
redstone.tools is the domain name right?

Reply:
Well yeah but i think "Redstone Tools" looks cleaner for a title